client/components/ManageTestQueue/ManageAtVersions.jsx

```diff
@@ -0,0 +1,451 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Form } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEdit, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import { DisclosureContainer } from '@components/ManageTestQueue/index';
+import BasicModal from '@components/common/BasicModal';
+import UpdateVersionModal from '@components/common/UpdateVersionModal';
+import { convertStringToDate } from '@client/utils/formatter';
+import { useMutation } from '@apollo/client';
+import {
+    ADD_AT_VERSION_MUTATION,
+    DELETE_AT_VERSION_MUTATION,
+    EDIT_AT_VERSION_MUTATION
+} from '@components/TestQueue/queries';
+import { useTriggerLoad } from '@components/common/LoadingStatus';
+import { THEMES, useThemedModal } from '@client/hooks/useThemedModal';
+import PropTypes from 'prop-types';
+
+const ManageAtVersions = ({ ats = [], triggerUpdate = () => {} }) => {
+    const { triggerLoad } = useTriggerLoad();
+    const {
+        themedModal,
+        showThemedModal,
+        setShowThemedModal,
+        setThemedModalTitle,
+        setThemedModalContent,
+        setThemedModalType,
+        setThemedModalActions,
+        setThemedModalShowCloseAction,
+        focus,
+        setFocusRef,
+        hideThemedModal
+    } = useThemedModal({
+        type: THEMES.WARNING,
+        title: 'Error Updating Assistive Technology Version'
+    });
+
+    const loadedAts = useRef(false);
```

Comment:
Is this used to prevent state updates on initial render? Just noting that this struck me as odd.

Reply:
Yep. This was originally moved from `ManageTestQueue/index.js` and there was an attempt to remove it, but found that re-renders were being significantly caused with several actions so opted to just restore that for now.